src/cluster/is_operator_workflow.go

```diff
@@ -0,0 +1,47 @@
+// Copyright © 2021 Banzai Cloud
```

Comment:
if I remember right we agreed we don't need this, so please remove it, or just update this to use the correct activity that we use in the other workflow